pandas/tests/io/parser/common/test_read_errors.py

```diff
@@ -32,10 +34,17 @@ def test_empty_decimal_marker(all_parsers):
     msg = "Only length-1 decimal markers supported"
     parser = all_parsers
 
+    if parser.engine == "pyarrow":
+        msg = (
+            "only single character unicode strings can be "
+            "converted to Py_UCS4, got length 0"
+        )
+
     with pytest.raises(ValueError, match=msg):
         parser.read_csv(StringIO(data), decimal="")
 
 
+@skip_pyarrow
```

Comment:
Why do a lot of these tests need to be skipped now?

Reply:
Prior to this PR, _all_ the tests in the file were skipped (using the `pytestmark` global variable) 
I've un-skipped the tests that now pass since the feature was added. Since I removed the global skip annotation, the tests that don't apply to pyarrow had to be individually annotated .